tidy3d/components/simulation.py

```diff
@@ -3596,6 +3619,9 @@ def _post_init_validators(self) -> None:
         self._validate_custom_source_time()
         self._validate_mode_object_bends()
         self._warn_mode_object_pml()
+        _validate_min_grid_spacing(
+            self.grid, threshold=1e-7, msg_prefix=f"'{self.__class__.__name__}'"
+        )
```

Comment:
Let's do 1e-6, even that sounds too large, but to be consistent with the uniform grid validator. Or maybe we should increase the threshold for both? I don't know if I missed anything in the other discussion but wouldn't even 1e-5 be more than reasonable (hard to imagine any simulation that needs to go even below 1e-4). @yaugenst ?